### DISCUSSION TITLE
How to exclude matches from response body

### DISCUSSION THREAD

**Discussion by hktalent** [projectdiscovery/nuclei#1869] [Q&A] [answered]
如何在yaml中编写
body中即要有包含的，同时也有需要排除的情况呢？

how to write in yaml
There must be included in the body, and there are also situations that need to be excluded?

thanks

**Comment by ehsandeep** [comment, marked as answer]
@hktalent you can use `negative` matcher to exclude any matches from body, for example: [iot/xp-webcam.yaml](https://github.com/projectdiscovery/nuclei-templates/blob/f6e52a6739030efdcf79c8da132af038cfdc435c/iot/xp-webcam.yaml#L26)
